### DISCUSSION TITLE
Line chart for data with large differences

### DISCUSSION THREAD

**Discussion by praj18** [recharts/recharts#2735] [Q&A]
I have a series of data which I would like to display in chart format, either by using line chart or composed chart (both bar and line).
However, the data that is generated is usually miles apart in terms of value. For example dataKey A can be in the 10s range, whilst dataKey B is in the 100,000s range, and as a result, dataKey A becomes barely visible on the chart.

So, is there a way I can dynamically display the points on the y-axis in a non-linear way?

A y-axis that is similar to this

![7jwJX](https://user-images.githubusercontent.com/62665365/145967528-abdb6f67-94e2-4fac-92ca-73153cbf4795.png)

**Comment by nikolasrieble** [comment]
Could you provide a toy example with reproducible code in the manner of a good stackoverflow question? Then I would be willing to investigate a solution.
